src/main/java/com/atlassian/bitbucket/jenkins/internal/client/exception/BitbucketClientException.java

```diff
@@ -31,6 +31,22 @@ public BitbucketClientException(IOException e) {
         body = null;
     }
 
+    public BitbucketClientException(String message) {
+        super(message);
+        responseCode = -1;
+        body = null;
+    }
+
+    public BitbucketClientException(String message, Throwable cause, String body) {
+        super(message, cause);
```

Comment:
please chain this constructor instead. Only one constructor should do stuff.. so just call `this(message, -1, body, cause)` (you may need to add this constructor)